en/docs/observability-and-monitoring/supported-observability-tools-and-platforms/opensearch.md

```diff
@@ -0,0 +1,186 @@
+# Observe metrics, traces, and logs using OpenSearch
+
+The sample [shop service](/observability-and-monitoring/supported-observability-tools-and-platforms/overview/#example-observing-a-sample-integration-service) will be used in this guide. Follow the steps given below to observe BI tracing, metrics, and logs in [OpenSearch](https://opensearch.org/).
+
+## Step 1 - Set up OpenSearch Deployment
+
+* Main components:
+      * Sample Service: service application that is being observed.
+      * OpenSearch: stores and indexes logs/traces; exposes API and metrics ports.
+      * OpenSearch Dashboards: visualizes data from OpenSearch via web UI.
+      * Data Prepper: receives and processes OpenTelemetry data, sends it to OpenSearch.
+      * Fluent Bit: collects app logs and forwards them for indexing.
+      * Setup Container: automates initial setup — creates index templates and imports dashboards.
+
+* Download and unzip the [opensearch-observability-dashboard.zip](https://github.com/ballerina-platform/module-ballerina-observe/releases/download/v1.0.0-opensearch-dashboard/opensearch-observability-dashboard.zip) in your local machine.
+
+* The structure of the `opensearch-observability-dashboard` directory is as follows.
+
+```
+    .
+    |____config
+    | 	    |____ballerina
+    | 	    |     |____Config.toml
+    |       |____dashboards
+    |       |   	|____opensearch_dashboards.yml
+    |       |____data-prepper
+    | 	    |     |____pipelines.yaml
+    |       |____fluent-bit
+    | 	    |      |____fluent-bit.conf
+    | 	    |      |____parser.conf
+    |       |      |____scripts
+    |       |            |____scripts.lua
+    |       |____.env
+    |
+    |____logs
+    |	      |____ballerina
+    |
+    |____setup
+    | 	    |____opensearch-dashboards-template.ndjson
+    | 	    |____index-template-request.json
+    |
+    |____docker-compose.yml
+```
+
+* Update **OPENSEARCH_INITIAL_ADMIN_PASSWORD** in the `path/to/opensearch-observability-dashboard/config/.env` file.
+
+```
+OPENSEARCH_INITIAL_ADMIN_PASSWORD=<PASSWORD> # Password for the OpenSearch admin user
+```
+
+This password will be used to access the OpenSearch server.
+
+* Navigate to the `path/to/opensearch-observability-dashboard` directory and run the following `docker compose` command in the terminal to start the OpenSearch deployment along with the BI application.
+
+```
+docker compose -f docker-compose.yml up -d
+```
+
+## Step 2 - Set up BI application for observability
+
+* Navigate to **file explorer** view and add the following to `main.bal`.
+
+      ```ballerina
+      import ballerinax/metrics.logs as _;
+      import ballerinax/jaeger as _;
+      ```
+
+* Navigate to **file explorer** view and create the `Config.toml` file in the package directory to set the runtime configurations as follows.
+
+   ```toml
```

Comment:
Seems this code is not indented. Pls fix it. 
<img width="854" height="661" alt="image" src="https://github.com/user-attachments/assets/423bbfbf-2e41-4559-8370-5cf01164f963" />

Reply:
Fixed with https://github.com/wso2/docs-bi/pull/121